### ISSUE TITLE
loss is always the same,dont smaller 

### ISSUE THREAD

**Issue by henbucuoshanghai** [endernewton/tf-faster-rcnn#455]
iter: 47220 / 70000, total loss: 0.354476
 >>> rpn_loss_cls: 0.000126
 >>> rpn_loss_box: 0.000068
 >>> loss_cls: 0.003523
 >>> loss_box: 0.000122
 >>> lr: 0.001000
speed: 0.276s / iter
iter: 47240 / 70000, total loss: 0.355023
 >>> rpn_loss_cls: 0.000094
 >>> rpn_loss_box: 0.000046
 >>> loss_cls: 0.001107
 >>> loss_box: 0.003153
 >>> lr: 0.001000

speed: 0.276s / iter
iter: 57020 / 70000, total loss: 0.353910
 >>> rpn_loss_cls: 0.000007
 >>> rpn_loss_box: 0.000003
 >>> loss_cls: 0.005448
 >>> loss_box: 0.000084
 >>> lr: 0.000100
speed: 0.276s / iter
iter: 57040 / 70000, total loss: 0.349502
 >>> rpn_loss_cls: 0.000060
 >>> rpn_loss_box: 0.000010
 >>> loss_cls: 0.000791
 >>> loss_box: 0.000273
 >>> lr: 0.000100
speed: 0.276s / iter

iter: 69960 / 70000, total loss: 0.347819
 >>> rpn_loss_cls: 0.000047
 >>> rpn_loss_box: 0.000001
 >>> loss_cls: 0.000157
 >>> loss_box: 0.000086
 >>> lr: 0.000100
speed: 0.275s / iter
iter: 69980 / 70000, total loss: 0.347693
 >>> rpn_loss_cls: 0.000048
 >>> rpn_loss_box: 0.000004
 >>> loss_cls: 0.000082
 >>> loss_box: 0.000033
 >>> lr: 0.000100
speed: 0.275s / iter
iter: 70000 / 70000, total loss: 0.348334
 >>> rpn_loss_cls: 0.000065
 >>> rpn_loss_box: 0.000004
 >>> loss_cls: 0.000523
 >>> loss_box: 0.000217
 >>> lr: 0.000100
speed: 0.275s / iter

**Comment by henbucuoshanghai** [comment]
at first steps,loss is 0.35     and always 0.35,0.34 in 70000steps,      why????

**Comment by henbucuoshanghai** [comment]
lr  is too big??

**Comment by BertramSun** [comment]
loss_cls: 0.003523
loss_box: 0.000122
is small enough or overfitting

**Comment by henbucuoshanghai** [comment]
but  at first 20 steps，the loss is 0.35 too, it does not change ............................

**Comment by henbucuoshanghai** [comment]
small enough     ....  ok
try   bigger..

**Comment by CRmost** [comment]
I have the same problem ,and the loss_cls and loss_box are the 0 .why??

**Comment by CRmost** [comment]
Is there anyone can tell me the reason why the loss_cls and loss_box are zero all  the time when i trained???
